blog/Migration-to-Fable2.md

```diff
@@ -0,0 +1,160 @@
+- title: Migration to Fable 2
+- subtitle: Everything you need to know to migrate
+
+## TL;DR
+
+With this document we are going to convert a Fable 1 project into a Fable 2 project.
+
+<div class="message is-info">
+<div class="message-body">
+This guide has been writed by converting Fulma.Minimal template from Fable 1 to Fable 2.
+
+You can see all the changes made by looking at [this commits](https://github.com/MangelMaxime/Fulma/commit/24c9269fc08eac9c66e774aec4a5fe4132968805).
+</div>
+</div>
+
+## Upgrade to Babel 7
+
+During Fable 2, development [Babel 7](https://babeljs.io/blog/2018/08/27/7.0.0) has been released so we need to upgrade to it. Hopefully for us, they created a tool to help us.
+
+1. Install [npx](https://github.com/zkat/npx#readme), this a CLI program allowing us to run npm packages.
+
+    - For npm: `npm install -g npx`
+    - For yarn: `yarn global add npx`
+
+2. Run `npx babel-upgrade --write`, this will modify the nearest `package.json` and convert all you old `babel-` reference to their scoped reference if their exist.
+
+    For example, `babel-polyfill` is converted to `@babel/polyfill`
+
+## Upgrade fable's npm package
+
+1. You need to upgrade `fable-loader` to latest version `2.x.x`
+
+    - For npm: `npm update fable-loader@latest`
+    - For yarn: `yarn upgrade fable-loader@latest`
+
+2. We don't need anymore `fable-utils`, so remove it
+
+    - For npm: `npm uninstal fable-utils`
+    - For yarn: `yarn remove fable-utils`
+
+
+## Update your webpack.config.js
+
+<div class="message is-info">
+<div class="message-body">
+
+In order to best benefit, of new Fable 2 tree-shaking and module size reduction we encourage you to upgrade all your webpack related package to their latest version.
```

Comment:
Comma and packages plural: In order to best benefit of new Fable 2 tree-shaking and module size reduction we encourage you to upgrade all your webpack related packages to their latest version.